composer.json

```
@@ -13,19 +13,22 @@
         "bin-dir": "bin"
     },
     "require": {
-        "php": "^7.4",
+        "php": ">=8.0",
         "predis/predis": "^1.1.1",
         "doctrine/cache": "~1.3",
-        "b1rdex/predis-compressible": "^0.5.0"
+        "b1rdex/predis-compressible": "^1.0",
+        "symfony/cache": "~5.0 || ~6.0",
+        "symfony/config": "~5.0 || ~6.0",
+        "symfony/dependency-injection": "~5.0 || ~6.0",
+        "symfony/event-dispatcher-contracts": "~5.0 || ~6.0",
+        "symfony/http-foundation": "~5.0 || ~6.0",
+        "symfony/http-kernel": "~5.0 || ~6.0"
```

if we support `5.0` then we also support `4.4`

```suggestion
        "symfony/cache": "~4.4 || ~5.0 || ~6.0",
        "symfony/config": "~4.4 || ~5.0 || ~6.0",
        "symfony/dependency-injection": "~4.4 || ~5.0 || ~6.0",
        "symfony/event-dispatcher-contracts": "~4.4 || ~5.0 || ~6.0",
        "symfony/http-foundation": "~4.4 || ~5.0 || ~6.0",
        "symfony/http-kernel": "~4.4 || ~5.0 || ~6.0"
```